app/data_model/answer_store.py

```diff
@@ -4,7 +4,12 @@
 
 
 class Answer(object):
-    def __init__(self, group_id, block_id, answer_id, value, group_instance=0, answer_instance=0):
+    def __init__(self, group_id=None, block_id=None, answer_id=None, value=None, group_instance=0, answer_instance=0):
+        valid = (group_id or answer_id or block_id or value) is not None
```

Comment:
Ah! I got it backwards, I thought "Missing one of" meant any one of them was missing causing it to be invalid, perhaps it could say "At least one of ... must be set"